includes/widgets/class-wp-job-manager-widget-recent-jobs.php

```diff
@@ -100,7 +105,7 @@ public function widget( $args, $instance ) {
 
 				<?php while ( $jobs->have_posts() ) : $jobs->the_post(); ?>
 
-					<?php get_job_manager_template_part( 'content-widget', 'job_listing' ); ?>
+					<?php get_job_manager_template( 'content-widget-job_listing.php', array( 'logo' => $instance['show_logo'] ) ); ?>
```

Comment:
Why are we making this change from `get_job_manager_template_part` to `get_job_manager_template` ? 
It seems it's because of the $args parameter not being present. In your PR message (or on the issue) it would be good to go over the decision for that and any other information. For example did you consider other solutions to this.

In regards to $instance['show_logo'], is that always going to be set? If it's not set, will it trigger a PHP notice?

Reply:
`get_job_manager_template_part` doesn't support passing arguments to the template.
In this context, we need to pass `$show_logo` variable to the tempate file in order to show/hide logo.
So, in order to pass the variable we need to use `get_job_manager_template` which supports passing `$instance['show_logo']`